src/python_testing/TC_AccessChecker.py

```diff
@@ -18,6 +18,10 @@ class AccessTestType(Enum):
     WRITE = auto()
 
 
+def is_global(id: int) -> bool:
```

Comment:
nit: is global what? is_global_attribute_id?

Reply:
another nit - function not used :) . I just removed it.